src/GoTrueAdminApi.ts

```diff
@@ -194,10 +194,14 @@ export default class GoTrueAdminApi {
       const total = response.headers.get('x-total-count') ?? 0
       const links = response.headers.get('link')?.split(',') ?? []
       if (links.length > 0) {
+        const regex = /page=(\d+)(?=&)[^>]*>; rel="(\w+)"/
```

Comment:
Actually would you mind pulling this regex up to the top scope, so that a new object is not being created every time the function is called?

Also, is it possible to not use a regex here at all? Maybe use `URLSearchParams` instead?